report-job-status/action.yml

```diff
@@ -79,8 +79,8 @@ runs:
             failedStepsMsg += failedSteps
           }
           
-          let message = `🔴 Job failed at <a href="${baseUrl}/${{ github.repository }}/">${{ github.repository }}</a>:
-          <b>Job</b>: <a href="${baseUrl}/${{ github.repository }}/actions/runs/${{ github.run_id }}">${{ github.job }}</a>
+          let message = `🔴 Job failed: <a href="${baseUrl}/${{ github.repository }}/actions/runs/${{ github.run_id }}">${{ github.job }}/${{ github.run_id }}</a>
+          <b>Repo</b>: <a href="${baseUrl}/${{ github.repository }}/">${{ github.repository }}</a>
           <b>Commit</b>: <a href="${baseUrl}/${{ github.repository }}/commit/${{ github.sha }}">${{ github.sha }}</a>
```

Comment:
Could you please put the `Commit` field after `Branch/Tag/Pull request`?

Reply:
sure